utils/utils.go

```diff
@@ -154,9 +154,6 @@ func ParseInputFile(r io.Reader) []string {
 	for scanner.Scan() {
 		totalLines++
 		universalURL := strings.TrimSpace(scanner.Text())
-		if universalURL == "" {
```

Comment:
if you remove this, there might be empty strings in temp

Reply:
yes, I know. But now that you can choose what urls you want this should be a problem. Also if I think about a file filled with url there shouldn't be any empty lines anyway?